tox.ini

```diff
@@ -1,9 +1,12 @@
 [tox]
 envlist =
-    flake8
-    py{38,39,310}-dj{32,41}-wt{41,51,52}
-    py311-dj41-wt{41,51,52}
+    py{38,39}-dj{32}-wt{41,51,52}
+    py310-dj42-wt{51,52}
     py311-dj42-wt{51,52}
```

Comment:
The main issue here is that django 3.2 doesn't like python 3.10 and django 41 also has issues with python3.11.